link-checker.js

```diff
@@ -0,0 +1,65 @@
+import graph from "./public/graph.json" with { type: "json" };
+import { parse } from "node-html-parser";
+import fs from "fs/promises";
+
+const report = {};
+
+for (const page of graph) {
+  const { outputHref, route } = page;
+  const html = await fs.readFile(new URL(outputHref), "utf-8");
+  const root = parse(html);
+  const links = root.querySelectorAll("a");
+
+  links.forEach((link) => {
+    if (!route.startsWith("/blog/") && link.getAttribute("href").startsWith("/")) {
+      const linkUrl = new URL(`https://www.greenwoodjs.dev${link.getAttribute("href")}`);
```

Comment:
Nit pick.. the `URL()` class can take 2 arguments, which would avoid the string concat.

```suggestion
      const linkUrl = new URL(link.getAttribute("href"), 'https://www.greenwoodjs.dev');
```

https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#parameters

Reply:
Oh right, I always forget about that lol.  Next time!  (Or happy to accept a PR for it 👍)